### DISCUSSION TITLE
Feature Request: iFrames (embed YouTube videos)

### DISCUSSION THREAD

**Discussion by NoodleNodes** [notaapp/talk#275] [Feature Requests]
I have several notes that include iFrames, mostly from web articles converted to markdown using GoodLinks, or directly from YouTube. This feature would be greatly appreciated in addition to GIF’s.

**Comment by astoilkov** [comment]
We actually did a survey recently for embedding online content. It was in third place so we will probably implement this in the future.

Thanks!

**Reply by NoodleNodes** [reply]
> We actually did a survey recently for embedding online content. It was in third place so we will probably implement this in the future.

It has become an important feature that many established apps tend to overlook. 

It is frustrating how slow some app developers are to adopt features widely used by writers, thinkers and researchers. For example, I have been using the Zettelkasten Method long before it erupted into the mainstream (circa 2013 I believe). As a writer, Ulysses was my go to, but my requests for [[wiki-links]] have been no-go to this day, as such Ulysses is no longer my writing environment. iA Writer has finally rolled them out, but a little too late for me, I moved on. Professional writing apps have evolved rapidly within the last 1-2 years, it is no longer sufficient to provide a beautiful writing tool, it must be a thinking tool also. Writers need all the help we can get to manage and wrangle the random thoughts and information rattling around in our minds. We need an app that strikes a balance between ordered structure and creative chaos. I have high hopes for Nota, it is the first app I have come across that seems to be on a forward and upward trajectory regarding being a professional writing environment and tool. My fingers are crossed for your success finger🤞it's time a dark horse shakes things up a little in this overcrowded arena.

**Reply by astoilkov** [reply]
We will do our best. ❤️
